modules/lsfilter/src/js/LSFilterList.js

```diff
@@ -632,17 +632,19 @@ function lsfilter_list(config)
 	};
 
 	this.banner_in_listview = function()
-	{
-		if($('div#nachos-page-banners').length > 0) {
-			var thead =  $(this.config.table).find('thead');
-			var header = $(thead).filter(function(){return !$(this).hasClass('floating-header');});
-			var banner = $('div#nachos-page-banners');
-			if(self.request_query.indexOf('recurring_downtimes') == -1){
-				banner.css('margin', '25px 0 3px 0');
-				header.remove();
-			}else{
-				banner.css('margin', '0px 0 3px 0');
-			}
-		}
-	}
+    {
+        var currentLocation = window.location.href.toString();
+
+        if($('div#nachos-page-banners').length > 0) {
+            var thead =  $(this.config.table).find('thead');
+            var header = $(thead).filter(function(){return !$(this).hasClass('floating-header');});
+            var banner = $('div#nachos-page-banners');
+            banner.css('margin', '25px 0 3px 0');
+            header.remove();
+        }
+
+        if(currentLocation.indexOf('hosts') === -1) {
```

Comment:
I like it! Good job!

One idea I got just right now was that maybe you could move this if-statement into the if statement above. Since the if-statement above is looking if #nachos-page-banners exists.
Then you could also reuse the variable and write `banner.css('margin', '0px 0px');` instead